tests/backends/models.py

```diff
@@ -111,3 +112,7 @@ class Author(models.Model):
 
 class Book(models.Model):
     author = models.ForeignKey(Author, models.CASCADE, to_field='name')
+
+
+class ReturningModel(models.Model):
+    created = models.DateTimeField(default=Now, returning=True, editable=False)
```

Comment:
Looks really cool!
Don't we need to document this argument? Here, for example: https://docs.djangoproject.com/en/2.1/ref/models/fields/#field-options

Or documentation comes somehow separately?

Reply:
I actually removed the documentation. I don't want this to be part of the public API. At least not for now where there is no use case for it.

Reply:
Is any test coverage lost if these tests are removed? As I said previously, "I think it would be best to remove the PostgreSQL tests that suggest the change is enabling some new behavior (which as you say, shouldn't be used outside of Django itself)." Already we have someone looking at the test and wanting to use the functionality. :-)

Perhaps a comment should be added to ` django/db/models/fields/__init__.py` to mark `returning` as private.

Reply:
@timgraham if this is about adding more models to the test suite. I can try to remove it in a later PR.